lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_action.rb

```diff
@@ -82,6 +89,9 @@ def self.run(params)
         if upload_status_response.binary_download_uri
           UI.message("🔗 Download the release binary (link expires in 1 hour): #{upload_status_response.binary_download_uri}")
         end
+
+        Actions.lane_context[:FIREBASE_APP_DISTRO_RELEASE] = release
+        release
```

Comment:
love your work here, was thinking to do similar things
what do you think if we return `upload_status_response` here instead of just `release` ?
so something like `testing_url` and other things are accessible

Reply:
Thanks for the comment @Rezisn! 

+1 to what @kaibolay said. All the useful information is part of the release, so decided not to return the entire `upload_status_response`.